src/hosted-buttons/index.js

```diff
@@ -0,0 +1,62 @@
+/* @flow */
+
+import { getMerchantID } from "@paypal/sdk-client/src";
+
+import { getButtonsComponent } from "../zoid/buttons";
+
+import {
+  buildHostedButtonCreateOrder,
+  buildHostedButtonOnApprove,
+  getHostedButtonDetails,
+  renderForm,
+} from "./utils";
+import type {
+  HostedButtonsComponent,
+  HostedButtonsComponentProps,
+  HostedButtonsInstance,
+} from "./types";
+
+export const getHostedButtonsComponent = (): HostedButtonsComponent => {
+  function HostedButtons({
+    hostedButtonId,
+  }: HostedButtonsComponentProps): HostedButtonsInstance {
+    const Buttons = getButtonsComponent();
+    const render = (selector) => {
+      // The SDK supports mutiple merchant IDs, but hosted buttons only
+      // have one merchant id as a query parameter to the SDK script.
+      // https://github.com/paypal/paypal-sdk-client/blob/c58e35f8f7adbab76523eb25b9c10543449d2d29/src/script.js#L144
+      const merchantId = getMerchantID()[0];
```

Comment:
should we throw an error here if multiple merchant id's are provided instead of silently ignoring it

Reply:
yes! check it out 4e1cef71